Jan 12 IIPC talk slides + save-your-threads > thread-keeper (#415)
harvard-lil/website-static: app/_posts/2023-01-18-thread-keeper-iipc-webinar.md

---
title: "IIPC Technical Speaker Series: Archiving Twitter"
author: matteo-cargnelutti
excerpt_separator: <!--more-->
---

> I have been invited by the [International Internet Preservation Consortium (IIPC)](https://netpreserve.org/) 

Reviewer: ```suggestion
I was invited by the [International Internet Preservation Consortium (IIPC)](https://netpreserve.org/) 
```

Author: Good catch, thanks :) !
